x-pack/plugins/fleet/server/services/epm/elasticsearch/ingest_pipeline/install.ts

```diff
@@ -262,7 +262,7 @@ export async function ensureFleetFinalPipelineIsInstalled(
   };
   const res = await esClient.ingest.getPipeline(
     { id: FLEET_FINAL_PIPELINE_ID },
-    esClientRequestOptions
+    { ...esClientRequestOptions, meta: true }
   );
 
   const installedVersion = res?.body[FLEET_FINAL_PIPELINE_ID]?.version;
```

Comment:
I think we can do this?

```suggestion
    { ...esClientRequestOptions }
  );

  const installedVersion = res?.[FLEET_FINAL_PIPELINE_ID]?.version;
```

Reply:
No, because the `statusCode` from the response is used below

https://github.com/elastic/kibana/blob/629ccde3a6986b62001ab955275f2e334e99ab30/x-pack/plugins/fleet/server/services/epm/elasticsearch/ingest_pipeline/install.ts#L269-L270